client/src/Views/ReportsPage/AllShouts.vue

```diff
@@ -92,19 +112,40 @@ export default {
   }
 }
 
-.all-reports {
+.all-container {
   background-color: v.$main-bkgrnd;
+}
+
+.all-reports {
   display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: space-evenly;
   width: 100%;
 
   &__table {
-    margin: 1rem;
-    width: 60%;
+    width: 100%;
   }
   &__chart {
-    margin: 1rem, 1rem, 0, 1rem;
-    width: 50%;
+    margin: 1rem 1rem 0 1rem;
+    width: 100%;
+  }
+
+  .v-data-table__wrapper {
+    height: 288px;
```

Comment:
I think this should be 287px, so that the line isn't as thick...sorry, I've spent too much time looking at it on the users page 😂